version.go

```diff
@@ -1,4 +1,4 @@
 package gendoc
 
 // VERSION is the version of protoc-gen-doc being used.
-const VERSION = "1.3.2"
+const VERSION = "1.3.3"
```

Comment:
This is updated during the release of the new version. No need to change it here.